include/mata/nfa/nfa.hh

```diff
@@ -224,14 +199,6 @@ public:
      */
     Nfa& concatenate(const Nfa& aut);
```

Comment:
Is it in place or not? concatenate or concatenation?

Reply:
It is in-place with chaining. It concatenates `*this` with `aut` in-place (in place of `*this`). Therefore, it should be kept as `concatenate()`.